peps/pep-0756.rst

```diff
@@ -0,0 +1,359 @@
+PEP: 756
+Title: Add PyUnicode_Export() and PyUnicode_Import() C functions
+Author: Victor Stinner <vstinner@python.org>
+PEP-Delegate: C API Working Group
+Status: Draft
+Type: Standards Track
+Created: 13-Sep-2024
+Python-Version: 3.14
+
+.. highlight:: c
+
+
+Abstract
+========
+
+Add functions to the limited C API version 3.14:
+
+* ``PyUnicode_Export()``: export a Python str object as a ``Py_buffer``
+  view.
+* ``PyUnicode_Import()``: import a Python str object.
+
+In general, ``PyUnicode_Export()`` has a O(1) complexity: no memory
+copy is needed. See the :ref:`specification <export-complexity>` for
+cases when a copy is needed.
+
+
+Rationale
+=========
+
+PEP 393
+-------
+
+:pep:`393` "Flexible String Representation" changed string internals in
+Python 3.3 to use three formats:
+
+* UCS1: Unicode range [U+0000; U+00ff], 1 byte per character
+* UCS2: Unicode range [U+0000; U+ffff], 2 bytes per character
+* UCS4: Unicode range [U+0000; U+10ffff], 4 bytes per character
```

Comment:
Like movies, I think the first one was unnumbered? The others are hyphented:

https://en.wikipedia.org/wiki/Universal_Coded_Character_Set

```suggestion
* UCS: Unicode range [U+0000; U+00ff], 1 byte per character
* UCS-2: Unicode range [U+0000; U+ffff], 2 bytes per character
* UCS-4: Unicode range [U+0000; U+10ffff], 4 bytes per character
```